src/ExadelMentorship.WebApi/Middlewares/ExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,31 @@
+﻿using ExadelMentorship.BusinessLogic.Exceptions;
+using System.Net;
+
+namespace ExadelMentorship.WebApi.Middlewares
+{
+    public class ExceptionHandlingMiddleware : IMiddleware
+    {
+        public async Task InvokeAsync(HttpContext context, RequestDelegate next)
+        {
+            try
+            {
+                await next(context);
+            }
+            catch (NotFoundException ex)
+            {
+                context.Response.StatusCode = (int)HttpStatusCode.NotFound;
+                await context.Response.WriteAsync(ex.Message);
+            }
+            catch (FormatException ex)
```

Comment:
I tried googling to verify this catch case with the HTTP status code. Can you plz comment on this?